### PR TITLE
Update master branch for deployment

### PR DESCRIPTION
**Pull request by jmbrunskill**
## Description
Updates master with the latest data from develop

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+RXNAV_SERVICE_REST=REST
+
+# Auth service not currently used, but leaving this here for skeleton
+AUTHENTICATION_SERVICE_URL=http://localhost
+AUTHENTICATION_SERVICE_PORT=9990
+AUTHENTICATION_SERVICE_REALM=auth/realms/unified-codes
+AUTHENTICATION_SERVICE_AUTH=protocol/openid-connect/token
+AUTHENTICATION_SERVICE_CLIENT_ID=unified-codes-data
+AUTHENTICATION_SERVICE_CLIENT_SECRET=asecretthing
+AUTHENTICATION_SERVICE_GRANT_TYPE=password
```
Comment (lache-melvin) [inline .env:16]: Oh hm, perhaps we ought to remove these... we're not going to use them

**Review by lache-melvin — APPROVED** [review]
🚀

File: backend/server/debian/scripts/backup.sh [hunk 2]
```diff
@@ -1,50 +1,46 @@
...
   echo "Unable to complete backup"
   error_handler
 fi
 
-#Right now we're not pruning by default as we have a lot of space via Backblaze turn this on to save storage (could take a while to run!)
-#echo "pruning"
-#/usr/bin/restic -q prune
-
 curl -s $CRONITOR_URL?state=complete
-echo "Completed"
```
Comment (lache-melvin) [inline backend/server/debian/scripts/backup.sh]: intentional?

Comment (jmbrunskill) [reply backend/server/debian/scripts/backup.sh]: Fixed :)

**Pull request merged**